fe/fe-core/src/main/java/com/starrocks/load/ExportChecker.java

```diff
@@ -147,4 +153,34 @@ private void runExportingJobs() {
             }
         }
     }
+
+    private boolean checkBeStatus(ExportJob job) {
```

Comment:
This function is a little misleading, it says check status, but also change the job status.

Reply:
done